config/default.json

```diff
@@ -51,5 +51,10 @@
     },
     "nelsontenantmanagementservicetack": {
         "tenanttable": "nelson-tenants"
+    },
+    "rdssnapshotexportpipelinestack": {
+        "bucketname": "${databasename}-snapshot-backup",
```

Comment:
Don't include variable values within the config file. Call this as bucketnamepostfix and compute the variable value in the code.

Reply:
fixed